Annotate endpoints using WebSocketEnabled

darryl-apple opened the pull request:
#### d7bb1648698c9ab6aa80555ab8f6b9e74c53db56
<pre>
Annotate endpoints using WebSocketEnabled
<a href="https://bugs.webkit.org/show_bug.cgi?id=288761">https://bugs.webkit.org/show_bug.cgi?id=288761</a>
<a href="https://rdar.apple.com/145788512">rdar://145788512</a>

Reviewed by Sihui Liu.

Annotate endpoints using WebSocketEnabled

* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.cpp:
(WebKit::NetworkSocketChannel::sharedPreferencesForWebProcess const):
* Source/WebKit/NetworkProcess/NetworkSocketChannel.h:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.messages.in:

Canonical link: <a href="https://commits.webkit.org/298367@main">https://commits.webkit.org/298367@main</a>
</pre>
<!--EWS-Status-Bubble-Start-->
https://github.com/WebKit/WebKit/commit/ebb1ba7954cabef16432740bf0aa508cf568745e

| Misc | iOS, visionOS, tvOS & watchOS  | macOS  | Linux |  Windows | Apple Internal |
| ----- | ---------------------- | ------- |  ----- |  --------- | ------ |
| [✅ 🧪 style](https://ews-build.webkit.org/#/builders/38/builds/115268 "Passed style check") | [✅ 🛠 ios](https://ews-build.webkit.org/#/builders/131/builds/34967 "Built successfully") | [✅ 🛠 mac](https://ews-build.webkit.org/#/builders/138/builds/25465 "Built successfully") | [✅ 🛠 wpe](https://ews-build.webkit.org/#/builders/5/builds/121369 "Built successfully") | [✅ 🛠 win](https://ews-build.webkit.org/#/builders/59/builds/65875 "Built successfully") | [✅ 🛠 ios-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/562a85e2-88bd-4598-8f26-6afdfbe94715) 
| | [✅ 🛠 ios-sim](https://ews-build.webkit.org/#/builders/130/builds/35621 "Built successfully") | [✅ 🛠 mac-AS-debug](https://ews-build.webkit.org/#/builders/123/builds/43535 "Built successfully") | [✅ 🧪 wpe-wk2](https://ews-build.webkit.org/#/builders/34/builds/87578 "Passed tests") | [  ~~🧪 win-tests~~](https://ews-build.webkit.org/#/builders/60/builds/42319 "The change is no longer eligible for processing. Pull Request was already closed when EWS attempted to process it.") | [✅ 🛠 mac-apple](https://ews-bridge.webkit.apple.com/builds/sw/T-276/92699e7b-e2c4-41dd-8b79-0fc7c00d5b52/d79815f5-5885-400b-9651-bc58563faa67) 
| [✅ 🧪 webkitperl](https://ews-build.webkit.org/#/builders/11/builds/118216 "Passed tests") | [✅ 🧪 ios-wk2](https://ews-build.webkit.org/#/builders/132/builds/28402 "Passed tests") | [✅ 🧪 api-mac](https://ews-build.webkit.org/#/builders/18/builds/103475 "Passed tests") | [✅ 🧪 api-wpe](https://ews-build.webkit.org/#/builders/41/builds/67974 "Passed tests") | | 
| | [✅ 🧪 ios-wk2-wpt](https://ews-build.webkit.org/#/builders/133/builds/27567 "Passed tests") | | [✅ 🛠 wpe-cairo](https://ews-build.webkit.org/#/builders/65/builds/65024 "Built successfully") | | 
| | [✅ 🧪 api-ios](https://ews-build.webkit.org/#/builders/13/builds/97787 "Passed tests") | [✅ 🧪 mac-wk2](https://ews-build.webkit.org/#/builders/136/builds/21712 "Passed tests") | [✅ 🛠 gtk](https://ews-build.webkit.org/#/builders/2/builds/124548 "Built successfully") | | 
| | [✅ 🛠 vision](https://ews-build.webkit.org/#/builders/128/builds/42219 "Built successfully") | [✅ 🧪 mac-AS-debug-wk2](https://ews-build.webkit.org/#/builders/122/builds/31597 "Passed tests") | [✅ 🧪 gtk-wk2](https://ews-build.webkit.org/#/builders/1/builds/96372 "Passed tests") | | 
| | [✅ 🛠 vision-sim](https://ews-build.webkit.org/#/builders/121/builds/42589 "Built successfully") | [✅ 🧪 mac-wk2-stress](https://ews-build.webkit.org/#/builders/8/builds/99665 "Passed tests") | [✅ 🧪 api-gtk](https://ews-build.webkit.org/#/builders/21/builds/96157 "Passed tests") | | 
| [✅ 🛠 🧪 merge](https://ews-build.webkit.org/#/builders/19/builds/24468 "Built successfully and passed tests") | [✅ 🧪 vision-wk2](https://ews-build.webkit.org/#/builders/126/builds/41378 "Passed tests") | [✅ 🧪 mac-intel-wk2](https://ews-build.webkit.org/#/builders/137/builds/19225 "Passed tests") | [  ~~🛠 playstation~~](https://ews-build.webkit.org/#/builders/134/builds/38169 "The change is no longer eligible for processing.") | | 
| | [✅ 🛠 tv](https://ews-build.webkit.org/#/builders/127/builds/42097 "Built successfully") | [✅ 🛠 mac-safer-cpp](https://ews-build.webkit.org/#/builders/120/builds/47647 "Built successfully") | | | 
| | [✅ 🛠 tv-sim](https://ews-build.webkit.org/#/builders/125/builds/41621 "Built successfully") | | | | 
| | [✅ 🛠 watch](https://ews-build.webkit.org/#/builders/129/builds/44944 "Built successfully") | | | | 
| | [✅ 🛠 watch-sim](https://ews-build.webkit.org/#/builders/124/builds/43347 "Built successfully") | | | | 
<!--EWS-Status-Bubble-End-->

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ccda6089cd81daad32398470fe8aa6c8ef798514. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/765da22ae4acc00f90b70ebb3742b7bede80bf1a. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/7879a2f684227359128183d10b2a62d2dc67c440. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/0180e0b2efb20f21e61566da2559abe76fde71fa. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/20ef0c257c9ad0008a4a261f1b6587cc7ec7ea0f. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/eca5a172c57d6ad3a0e803c6e83e4e54184d1b4a. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

webkit-early-warning-system commented:
Starting EWS tests for https://github.com/WebKit/WebKit/commit/ebb1ba7954cabef16432740bf0aa508cf568745e. Live statuses available at the PR page, https://github.com/WebKit/WebKit/pull/41556

darryl-apple committed:
Annotate endpoints using WebSocketEnabled
https://bugs.webkit.org/show_bug.cgi?id=288761
rdar://145788512

Reviewed by Sihui Liu.

Annotate endpoints using WebSocketEnabled

* Source/WebKit/NetworkProcess/NetworkConnectionToWebProcess.messages.in:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.cpp:
(WebKit::NetworkSocketChannel::sharedPreferencesForWebProcess const):
* Source/WebKit/NetworkProcess/NetworkSocketChannel.h:
* Source/WebKit/NetworkProcess/NetworkSocketChannel.messages.in:

Canonical link: https://commits.webkit.org/298367@main
